O/onnxruntime/build_tarballs.jl

```diff
@@ -0,0 +1,99 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "onnxruntime"
+version = v"1.10.0"
+
+# Collection of sources required to complete build
+source_sources = [
+    GitSource("https://github.com/microsoft/onnxruntime.git", "0d9030e79888d1d5828730b254fedc53c7b640c1")
+]
+
+# Bash recipe for building across all platforms
+source_script = raw"""
+cd $WORKSPACE/srcdir
+
+wget https://github.com/Kitware/CMake/releases/download/v3.22.1/cmake-3.22.1-linux-x86_64.sh
```

Comment:
We already have cmake, don't we?

Reply:
Yeah - I wrote this in early January for BB v0.4 - will update/fix.

Reply:
Fixed